frontend/packages/component-library/package.json

```diff
@@ -443,6 +544,7 @@
     "rimraf": "^6.0.1",
     "schema-dts": "^1.1.5",
     "shadow-dom-testing-library": "^1.13.1",
+    "storybook": "^9.0.18",
```

Comment:
Pretty sure these 3 do not need to be added here as they are in the `design-system-storybook`?